src/components/CreateCredential.js

```diff
@@ -17,53 +20,131 @@ const CreateCredential = () => {
+
+        <GetCredential.Card.Side>
+          <div style="display : flex ; gap : 16px ; flex-direction : column;">
             <h3 className="spectrum-Heading spectrum-Heading--sizeS side-header" >API key credential</h3>
             <p className="spectrum-Body spectrum-Body--sizeM">An API Key credential was created. The API key credential identifies your application to Adobe servers and can help accept or reject request originating from certain domains.</p>
             <h3 className="spectrum-Heading spectrum-Heading--sizeS side-header" >Learn more</h3>
-            <a className="side-documentation" href='https://developer.adobe.com/express/embed-sdk/docs/guides/quickstart/'>Quickstart guide</a>
-            <a className="side-documentation" href='https://developer.adobe.com/express/embed-sdk/docs/guides'>Adobe Express Embed SDK documentation</a>
+            <a className="side-documentation" style={{ color: "#0265DC" }} href='https://developer.adobe.com/express/embed-sdk/docs/guides/quickstart/'>Quickstart guide</a>
+            <a className='side-documentation' style={{ color: "#0265DC" }} href='https://developer.adobe.com/express/embed-sdk/docs/guides/'>Adobe Express Embed SDK documentation</a>
           </div>
-        </GetCredential.Side>
+        </GetCredential.Card.Side>
+
+        <GetCredential.Card.Products label="Included products and services">
+          <GetCredential.Card.Product label="Adobe Express Embed SDK" icon={creativeCloud} />
+        </GetCredential.Card.Products>
+
+        <GetCredential.Card.DevConsoleLink heading="Developer Console Project" />
+
+        <GetCredential.Card.CredentialDetails heading="Credential details">
+          <GetCredential.Card.CredentialDetails.APIKey heading="API Key" />
+          <GetCredential.Card.CredentialDetails.AllowedOrigins heading="Allowed domains" />
+          <GetCredential.Card.CredentialDetails.OrganizationName heading="Organization" />
+        </GetCredential.Card.CredentialDetails>
 
       </GetCredential.Card>
 
+      <GetCredential.Return title="Previously created projects" paragraph="Select a project and access your existing credentials for Firefly - Firefly and Creative Cloud Automation." nextStepsLabel="Next steps" nextStepsHref="/credentials/nextsteps" className="card_developer_console">
+
+        <GetCredential.Return.Side>
+          <GetCredential.Return.Side.Custom>
+            <div style={{ display: "flex", gap: "30px", flexDirection: "column", width: "100%" }}>
+              <h3 className='spectrum-Heading spectrum-Heading--sizeM'>Welcome back</h3>
+              <p className="spectrum-Body spectrum-Body--sizeM">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation.</p>
```

Comment:
We need to replace this placeholder text

Reply:
Yes

Reply:
@ErinFinnegan, Could you please share the placeholder ?